src/UI/Components/ConfirmUserActionForm/ConfirmUserActionForm.test.tsx

```diff
@@ -9,7 +9,9 @@ test("GIVEN DeleteForm WHEN user clicks 'No' THEN closeModal is executed", async
 
   render(<ConfirmUserActionForm onSubmit={submit} onCancel={closeModal} />);
 
-  await userEvent.click(screen.getByRole("button", { name: "No" }));
+  await act(async () => {
```

Comment:
I don't get it, based on the newest version of the library, userEvents shouldn't be wrapped anymore. Why do we need to wrap those then?